docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -15,7 +15,7 @@ type: docs
 Use the CREATE INDEX statement to create an index on the specified columns of the specified table. Indexes are primarily used to improve query performance.
 
 {{<note>}}
-In YugabyteDB, indexes are global and are implemented just like tables. They are split into tablets and distributed across the different nodes in the cluster. The sharding of indexes is based on the primary key of the index and is independent of how the main table is sharded and distributed. Indexes are not colocated with the base table.
+In YugabyteDB, indexes are global and are implemented just like tables. They are split into tablets and distributed across the different nodes in the cluster. The sharding of indexes is based on the primary key of the index and is independent of how the main table is sharded and distributed. Indexes are not colocated with the base table by default, but if the table is created colocated, then the index follows the table.
```

Comment:
```suggestion
In YugabyteDB, indexes are global and are implemented just like tables. They are split into tablets and distributed across the different nodes in the cluster. The sharding of indexes is based on the primary key of the index and is independent of how the main table is sharded and distributed.
```
This brings up "colocated" too suddenly. And you ought to bring up "copartitioning" too in that case. Instead, how about omit this detail. Otherwise, you could say "Unless the index is colocated or copartitioned, the sharding of indexes..."